templates/components/search/item_filter.html.twig

```diff
@@ -0,0 +1,81 @@
+{#
+ # ---------------------------------------------------------------------
+ #
+ # GLPI - Gestionnaire Libre de Parc Informatique
+ #
+ # http://glpi-project.org
+ #
+ # @copyright 2015-2023 Teclib' and contributors.
+ # @copyright 2003-2014 by the INDEPNET Development Team.
+ # @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ #
+ # ---------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of GLPI.
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+{# Display the Filter tab #}
+<div class="row">
+    {# Add a bit of padding with the container class #}
+    <div class="col-12 search-container container">
+        {# Info section detailling the purpose of the filter #}
+        <div class="bd-callout bd-callout-info mt-1">
+            <h3 class="fw-normal"><i class="ti ti-info-circle me-1"></i>{{ info_title }}</h3>
+            <p class="text-muted">{{ info_description }}</p>
+        </div>
+
+        {# Do not show filters until enabled #}
+        <div id="manage_filter_ux" class="{{ filter_enabled ? "" : "d-none" }}">
+
+            {# Display search UX #}
+            {{ call('Glpi\\Search\\Input\\QueryBuilder::showGenericSearch', [itemtype, params]) }}
```

Comment:
This method is outputing HTML directly and does not return a printable value, so you should not use `{{ ... }}` (it is made to print something).
```suggestion
            {% do call('Glpi\\Search\\Input\\QueryBuilder::showGenericSearch', [itemtype, params]) %}
```